README.md

```diff
@@ -618,6 +748,48 @@ class ApplicationJob < ActiveJob::Base
 end
 ```
 
+### Customizing the Database Models
+
+You can modify the database models by editing the generated migrations.
+
+One common adjustment is to change the IDs to UUIDs (if you're using UUIDs in your app).
+
+You can also add additional columns to the `Noticed::Event` and `Noticed::Notification` models. 
+
+```ruby
+# This migration comes from noticed (originally 20231215190233)
+class CreateNoticedTables < ActiveRecord::Migration[7.1]
+  def change
+    create_table :noticed_events, id: :uuid do |t|
+      t.string :type
+      t.belongs_to :record, polymorphic: true, type: :uuid
+      t.jsonb :params
+
+      # Custom Fields
+      t.string :organisation_id, type: :uuid, as: "((params ->> 'organisation_id')::uuid)", stored: true
+      t.virtual :action_type, type: :string, as: "((params ->> 'action_type'))", stored: true
+      t.virtual :url, type: :string, as: "((params ->> 'url'))", stored: true
```

Comment:
I had never thought of using virtual attributes to extract data from params. That's a great idea.